sdapi_ingest_scripts/configs/raven.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Raven.configure do |config|
+  config.dsn = ENV['SENTRY_DSN']
+  config.current_environment = Rails.env
+end
```

Comment:
Wondering if this might be useful to keep? Were these changes needed for the jobs to pick up errors in addition to the change in good_job.rb? 

Also, everything reports under production in Sentry as it currently is set up, so this should give us the environment breakdown. Or maybe check with April to see if there are other changes that could be made so it works right without it?

Reply:
Ok, I'll take some time to find out how sentry works. Rob said we shouldn't need this file at all so I wasn't sure about adding it directly to main.